src/extract/strategy_smart_by_first_node_and_tags.cpp

```diff
@@ -203,7 +218,7 @@ namespace strategy_smart_by_first_node_and_tags {
                         if (e->node_ids.get(member.positive_ref())) {
                             if (wanted_members == 0) {
                                 e->relation_ids.set(relation.positive_id());
-                                if (strategy().check_type(relation) && strategy().check_tags(relation)) {
+                                if (strategy().check_type(relation) || strategy().check_tags(relation)) {
```

Comment:
I am not sure why we are seeing this change here.
It was already done in 4213ca521c494aafb25dd3410c2902e6ec41f330.